scripts/go.mod

```diff
@@ -3,12 +3,15 @@ module scripts
 go 1.13
 
 require (
+	github.com/pingcap/errors v0.11.5-0.20200917111840-a15ef68f753d // indirect
 	github.com/pingcap/log v0.0.0-20210906054005-afc726e70354
-	github.com/pingcap/tidb-dashboard v0.0.0-00010101000000-000000000000
 	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
 	github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546
 	github.com/swaggo/swag v1.6.6-0.20200529100950-7c765ddd0476
 	go.uber.org/zap v1.19.0
+	golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5 // indirect
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
+	golang.org/x/tools v0.0.0-20210112230658-8b4aab62c064 // indirect
```

Comment:
Please remove all of these newly added indirect instructions. I guess they are not needed. If you find it hard to remove, follow these steps:

1. remove `go.sum`
2. remove all `// indirect` here
3. run `go mod tidy` to generate a new `go.sum`

I usually find it working with the steps above.

Reply:
Got it.